packages/firebase_analytics/firebase_analytics_web/README.md

```diff
@@ -0,0 +1,83 @@
+# firebase_analytics_web
+
+The web implementation of [firebase_analytics][1]
+
+## Usage
+
+### Import the package
+
+This package is the endorsed implementation of `firebase_analytics` for the web platform since version `5.0.15`, so it gets automatically added to your application by depending on `firebase_analytics: ^5.0.15`.
```

Comment:
I think it'll be endorsed in `5.0.16`, since we've already published 5.0.15 without endorsing this :)